portfolio/src/main/webapp/index.html

```diff
@@ -7,68 +7,77 @@
         <h1>Nicolas Perez Vergel</h1>
         <p> (call me Nico) </p>
       </div>
+
       <div id="personal-info">
-           <button onclick="addRandomFact()">Click for random fact</button>
-           <div id="fact"></div>
+        <button onclick="addRandomFact()">Click for random fact</button>
+        <div id="fact"></div>
       </div>
+
       <div id="Background">
-      <h1> Background: </h1>
-        <p> I am a rising junior at Harvey Mudd College. This summer I am interning at Google for the STEP program. I know Python, Java, C++ and Javascript. </p>
+        <h1> Background: </h1>
+        <p> I am a rising junior at Harvey Mudd College.
+            This summer I am interning at Google for the STEP program.
+            I know Python, Java, C++ and Javascript.
+        </p>
       </div>
+
       <div>
-      <h1> Personal Projects: </h1>
-        <button type="button" class="collapsible">Acceleration Scheme for Semi-Autonomous Vehicles</button>
-        <div class="content">
-        <p> Developed an algorithm to calculate the safest and most efficient acceleration scheme for a platoon. 
-            <a href="paper.html">Paper</a> currently under review. </p>
+        <h1> Personal Projects: </h1>
+        <button class="collapsible">Acceleration Scheme for Semi-Autonomous Vehicles</button>
+        <div class="content hidden">
+          <p> Developed an algorithm to calculate the safest and most efficient acceleration scheme for a platoon. 
+            <a href="paper.html">Paper</a> currently under review.
+          </p>
         </div>
-        <button type="button" class="collapsible"> App for buying and selling used textbooks </button>
-        <div class="content">
-        <p> In college I made an <a href="https://github.com/jgrant12345/Social-App-Textbooks">app</a> to trade textbooks since new ones are overly priced! </p>
+        <button class="collapsible"> App for buying and selling used textbooks </button>
+        <div class="content hidden">
+          <p> In college I made an <a href="https://github.com/jgrant12345/Social-App-Textbooks">app</a>
+              to trade textbooks since new ones are overly priced!
+          </p>
         </div>
-        <button type="button" class="collapsible">House Valuation</button>
-        <div class="content">
-        <p> Using machine learning we can estimate a price for your house (as long as you live in Fort Collins) <br>
-        <a href="http://focofoodie.com/"> FOCO House Pricing </a> </p>
+        <button class="collapsible">House Valuation</button>
+        <div class="content hidden">
+          <p> Using machine learning we can estimate a price for your house (as long as you live in Fort Collins) <br>
+            <a href="http://focofoodie.com/"> FOCO House Pricing </a>
+          </p>
         </div>
         <script>
-        collabsible()
+          collapsible() //This funcion adds a listener to .collapsble which hides/unhides .content
```

Comment:
nit: typo

Reply:
Also, consider moving this into your javascript file - this doesn't need to go in the HTML source.

If you're running into the issue of the script executing before the HTML is fully loaded you could Use an Onload listener to trigger this behavior.

In general we're going to try to avoid mixing types: javascript and html ought be as independent and decoupled as possible.

Reply:
I had decided not to put it in the onload because it is helps see what that function is specifically for since its under the elements it acts on. However if its bad syntax I will move it.